sct.py

```diff
@@ -496,8 +498,9 @@ def list_ami_versions(region):
               type=CloudRegion(cloud_provider="aws"),
               default='eu-west-1',
               help="a region to look for AMIs (default: eu-west-1)")
+@click.option('-a', '--arch', type=click.Choice(['x86_64', 'arm64']))
```

Comment:
it should have had a default here (and IMHO for now, it should default to `x86_64`)